### DISCUSSION TITLE
Bootstrap Email and SMTP server integration

### DISCUSSION THREAD

**Discussion by kumy** [bootstrap-email/bootstrap-email#108] [Show and tell]
Hi there,

for the need of my project, I've built an SMTP server around [midi-smtp-server](https://github.com/4commerce-technologies-AG/midi-smtp-server) which process the body through [Bootstrap Email](https://v1.bootstrapemail.com/) and then forward it to a smtp relay host. That way it should be easy for many projects to use Bootstrap Email without native language binding just by changing the smtp gateway… I'll use it with Fat Free Framework, and templating using Smarty 3.

Project is available here https://github.com/geokrety/bootstrap-email-smtp-server. It was inspired from [Slack recipe](https://github.com/4commerce-technologies-AG/midi-smtp-server/tree/master/cookbook/recipe-slack)

That's my first time using Ruby, I hope it's clean enough.

Enjoy
@kumy

**Comment by stuyam** [comment]
An SMPT server is a super interesting concept. Would be very cool for people to be able to just "deploy" a simple smtp server to something like Heroku or Digital Ocean or maybe something container based and have it work as a proxy. Very interesting I think there are some cool possibilities for this in the future and could allow for easier integration into other codebases.

Thanks giving this is a go and putting a project together, very cool!
